### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by pisuke**
Work in progress proposal to make the documentation easier to browse.

### CODE REVIEW

File: docs/readme.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+* UDMI definition
+* [Technology stack](tech_stack.md)
+* Message types
+  * metadata.json
+  * envelope.json
+  * state.json
+  * event.json
+  * config.json
+  * discover.json
```
Comment (grafnu) [inline docs/readme.md:13]: discover shouldn't be in the list -- it's a kind of event (like pointset)

File: docs/readme.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    * Java UDMI library
+    * Python UDMI library
+
+* UDMI applications
+  * registrar
+  * keygen 
+  * genkeys
+  * pubber
+  * validator
+  * sequence
```
Comment (grafnu) [inline docs/readme.md:38]: this is now "sequencer" for consistency!

**Review by grafnu — COMMENTED** [review]
two minor comments!

File: docs/readme.md (hunk 1 above)
Comment (grafnu) [reply docs/readme.md:13]: still rogue discover

File: docs/readme.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+* [Technology stack](tech_stack.md)
+* Message types
+  * metadata.json
+  * envelope.json
+  * state.json
+  * event.json
+  * config.json
+  * discover.json
+* UDMI site model
+* UDMI transactions
```
Comment (grafnu) [inline docs/readme.md:15]: nit: I think we should be consistent and use "sequences" -- (e.g. there is a bin/sequencer already for testing things)

**Comment by pisuke** [comment]
Closing this PR as it is now embedded in PR #210

**Pull request closed without merging**